group/assignment6/Move.h

```diff
@@ -0,0 +1,20 @@
+#ifndef MOVE_H
+#define MOVE_H
+#pragma once
+
+struct Move
+{
+	int carId;
+	int startPosition;

```

Comment:
I still believe that startPosition and endPosition should be treated as a pair of ints so that they would be separated from the carId. What do you think?

Reply:
I don't mind, so have a pair in structure? 

struct Move
{
	int carId;
	pair<int, int> positions;
This way? or you prefer vector of pairs of pairs without structure?

Reply:
Yes, this way is good for me, the structure is needed to keep code readable.